### ISSUE TITLE
3.2.1.2 Quizmaster prompts on a CVR

### ISSUE THREAD

**Issue by kclimenhaga** [gryphonshafer/Quizzing-Rule-Book#214]
Current text: _On a CVR/CVRMA, in the scenario where the entire reference (which is part of the question text) was not read by the quizmaster, the quizmaster has 2 prompts to give, and must give them in correct order and with correct timing. If the quizzer provides all information in the question text and the answer text, because the quizzer has not yet finished saying the complete reference, which is part of the question, the quizmaster must prompt with "More." The quizmaster cannot prompt with "What is your question?" yet, because the quizzer has not provided all information in the question (which includes the reference) and the answer._

Since the "More" prompt has been removed in all other places, should this be changed to "What is your reference"? 

In the RC minutes, it looks like this was discussed, but the final proposal for quizmaster prompts did not include any changes to the Commentary section, so I don't think we voted on it. Can this be revisited?

**Comment by jttower** [comment]
Yes, all the references to the old QM prompts like "more" should be removed or changed to what was already voted on.  I'm not sure if this needs to wait another year, I think it should just be changed right away to avoid confusion.  With the streamlining of prompts, the intent was to change them throughout the rule book, not just in the QM prompts section.

**Comment by JoshJetto** [comment]
Agreed. Since the commentary is merely explanatory material and nontge substantive rule, thus should fall under grammatical-type fixes and should not require a CQLT approval to make the change but can simply be done as an edit not a rule change.

**Comment by ARMediting** [comment]
That does seem like an oversight. Glad to see the new prompt rules finally official!

**Comment by kclimenhaga** [comment]
Okay, great! How exactly do we want to make this change? It isn't as straightforward as just removing the applicable section. My preference would be that if the quizzer has not yet said the full reference, but has said the rest of the question and answer, the quizmaster should prompt with "What is your question?", and then if the quizzer does not finish the reference, the quizmaster should prompt with "What is your reference?" (or similar). That is exactly opposite of what this section says, but to me any other order is confusing.
